contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -1459,6 +1459,159 @@ class tsqlBuilder : public tsqlCommonMutator
 		popContainer(ctx);
 	}
 
+	void exitAlter_table(TSqlParser::Alter_tableContext *ctx) override
+	{
+		if (ctx->TRIGGER() && ctx->id().size() > 0)	/* condition to filter alter table statements which contains TRIGGER keyword and multiple trigger names */
+		{
+			/*
+			 * When we come across a alter table query which enable/disable trigger with multiple trigger name, 
+			 * we will replace it with list of alter table statements, a statement for each trigger.
+			 * As Postgres only support enabling/disabling of only one trigger using alter table syntax, so a call like:
+			 * 
+			 * 	ALTER TABLE Employees { ENABLE | DISABLE } TRIGGER trigger_a, trigger_b
+			 *	GO
+			 * 
+			 * will be re-written as:
+			 * 
+			 * 	ALTER TABLE Employees { ENABLE | DISABLE } TRIGGER trigger_a; ALTER TABLE Employees { ENABLE | DISABLE } TRIGGER trigger_b;
```

Comment:
how do we handle failures here? what if for alter1; alter2; alter3 alter1 fails and alter2 fails.

Reply:
All these statements, alter1; alter2; alter3 are given as batch of statements, when any one of them fails, the whole query will fail.